src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IVariableDeclaration.cs

```diff
@@ -456,9 +456,11 @@ static void Main(string[] args)
 IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement) (Syntax: 'int* p = &reference.i1')
   IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration) (Syntax: 'p = &reference.i1')
     Variables: Local_1: System.Int32* p
-    Initializer: IAddressOfExpression (OperationKind.AddressOfExpression, Type: System.Int32*) (Syntax: '&reference.i1')
-        IFieldReferenceExpression: System.Int32 Program.i1 (OperationKind.FieldReferenceExpression, Type: System.Int32) (Syntax: 'reference.i1')
-          Instance Receiver: ILocalReferenceExpression: reference (OperationKind.LocalReferenceExpression, Type: Program) (Syntax: 'reference')
+    Initializer: IOperation:  (OperationKind.None) (Syntax: '&reference.i1')
```

Comment:
>IOperation:  (OperationKind.None) (Syntax: '&reference.i1') [](start = 17, length = 59)

OperationKind.None feels unexpected in this scenario.

Reply:
done - https://github.com/dotnet/roslyn/issues/22557